### PR TITLE
log errno names instead of values

### PR DESCRIPTION
**Pull request by udit043**

### CODE REVIEW

File: apps/telepathy/telepathy-resiprocate.cxx [hunk 1]
```diff
@@ -27,13 +27,18 @@
...
 #include "Protocol.hxx"
 
 using namespace tr;
 
 int main(int argc, char *argv[])
 {
+   //Creating error map for OS, SSL and X509 error numbers
+   ErrnoError::CreateMappingErrorMsg();
+   OpenSSLError::CreateMappingErrorMsg();
+   //X509Error::CreateMappingErrorMsg();
```
Comment (udit043) [inline apps/telepathy/telepathy-resiprocate.cxx]: I have not used any X509 Error message in the project, should I remove all the X509 error messages from Errdes.cxx or would it be needed in future ?

Comment (dpocock) [inline apps/telepathy/telepathy-resiprocate.cxx]: you can remove the X509 stuff

File: rutil/Errdes.cxx [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+map <int, string> OpenSSLErrorMsg;
+map <int, string> X509ErrorMsg;
+
+
+string ErrnoError::SearchErrorMsg(int Error)
+{
+  string result;
+
+#ifdef _WIN32
+    result = WinErrorMsg[Error];
```
Comment (dpocock) [inline rutil/Errdes.cxx]: check if the map contains the key before trying to retrieve the value

please do the same thing for each of the other lookups too

Comment (udit043) [inline rutil/Errdes.cxx]: done

**Comment by udit043** [comment]
Closing this pull request, the discussion continued on #164

**Pull request closed without merging**